gradle.properties

```diff
@@ -3,16 +3,16 @@ org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 # check these on https://modmuss50.me/fabric.html
     minecraft_version=1.19.2
-    yarn_mappings=1.19.2+build.1
-    loader_version=0.14.9
+    yarn_mappings=1.19.2+build.28
+    loader_version=0.14.10
 # Mod Properties
     mod_version=1.4.7
     maven_group=me.flashyreese.mods
     archives_base_name=reeses_sodium_options
 
 # Dependency
-    sodium_version=0.4.2+build.181
+    sodium_version=0.4.4+build.198
```

Comment:
Please revert this, this will break the dev env since Iris 1.2.5 is not compatible with that version.

Reply:
undoed.